calculator/src/App.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import Instructions from './components/Instructions';
+import Inputs from './components/Inputs';
+import Output from './components/Ouput'
+import './app.css';
+
+function App() {
+
+  const [total, setTotal] = useState(0);
+  const [tip, setTip] = useState(0);
+  const [data, setData] = useState({
+    total: 0,
+    tip: 0,
+    billTotal: 0
+  });
+
+
+
+  const calculateTip = (total, tip) => {
```

Comment:
This function is actually not calculating the tip, but passing the values to the parent. maybe something to take into account